packages/components/src/components/positions-drawer-card/positions-drawer-card.tsx

```diff
@@ -157,13 +163,13 @@ const PositionsDrawerCard = ({
         </React.Fragment>
     );
 
-    const supported_contract_card = (
+    const unsupported_contract_card = (
```

Comment:
I noticed, that actually names were swapped. It become clear, if you check line 212